.ct.yaml

```diff
@@ -0,0 +1,4 @@
+remote: origin
+target-branch: main
```

Comment:
How does this work when running on a pull request?

Reply:
From what I understand this is the base branch that is compared with the current branch. You can see in this PR that it detected changes and checked them.